js/common.js

```diff
@@ -231,12 +231,11 @@ $(document).ready( function() {
         enableDateTimePicker(this);
 	});
 
-	if( $( ".dropzone-form" ).length ) {
-		enableDropzone( "dropzone", false );
-	}
-	if( $( ".auto-dropzone-form" ).length ) {
-		enableDropzone( "auto-dropzone", true );
-	}
+	$( 'form .dropzone' ).each(function(){
+		var classPrefix = 'dropzone';
```

Comment:
I guess that `enableDropzone( classPrefix, autoUpload )` was written with the idea to allow different class names to match the dropzone elements. For example, allowing more than one DZ, for different forms, in the same page.
However, i think that the library is internally dependant on elements with "dropzone" class. At least, i couldn't make it work in other way.

So, the class name is still hardcoded, but i kept the class-prefix parameter in case we would want to improve that feature eventually.